app/modules/entities/components/layouts/entity_layout_actions.svelte

```diff
@@ -55,15 +54,8 @@
       <span class="button-text">{I18n('refresh Wikidata data')}</span>
     </button>
   </li>
-  <li>
-    <Link
-      url={wikidataHistoryUrl}
-      text={I18n('entity history')}
```

Comment:
Looks like there is still a need to have a link to Wikidata history page for non-local layer entities, ie `5/entity/wd:Q42511/history`

Reply:
Done in 7c67b070f